manager/integration/tests/test_system_backup_restore.py

```diff
@@ -259,7 +260,10 @@ def create_system_backup_and_assert_volume_backup_count(count):
         backup_volume = find_backup_volume(client, volume_name)
         wait_for_backup_count(backup_volume, count)
 
-    create_system_backup_and_assert_volume_backup_count(1)
+        return backup_volume
+
+    bv = create_system_backup_and_assert_volume_backup_count(1)
+    check_volume_last_backup(client, volume_name, bv.lastBackupName)
```

Comment:
Do you think it would make sense to move check_volume_last_backup into create_system_backup_and_assert_volume_backup_count?

Reply:
Yes, it could be moved into the `create_system_backup_and_assert_volume_backup_count`, and I will have an update soon. Thanks.